docs/cookbook/creating_content_with_binary_attachments_via_rest_api.md

```diff
@@ -1,18 +1,16 @@
 # Creating content with binary attachments via REST API
 
-## Description
+This page shows how to create content via the REST API on the example of uploading an image file.
```

Comment:
I removed all Solution and Description heading because:
1. They make the in-page table of contents useless (every cookbook recipe just has a "Description" and "Solution" heading, no way to quickly navigate thoughout the page)
2. Descriptions are always the first thing in the page, so to link to it you can just link to the page itself.
3. Descriptions are very short, so there's no need to make Solution a separate anchor, because in almost all pages you can instantly see it when you visit the page.
4. Not every page has a logical Description section

Reply:
1, 2 and 3 :ok: 

4. It's accurate to have a description of the case and solution for Recipes in the Cookbook, not on every page.

Reply:
And we still have descriptions and solutions, just not emphasised as separate headings/sections :)

Reply:
Because of points 1-3. above :) In short, in my opinion the pages will be better structured and easier to quickly browse for information if we use H2 for topic-specific headings.